gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -0,0 +1,790 @@
+            "custom_select": custom_joint_faf_select,
+            "select_globals": [
+                "joint_freq_meta",
+                "joint_freq_index_dict",
+                "joint_freq_meta_sample_count",
+                "joint_faf_meta",
+                "joint_faf_index_dict",
+            ],
+        },
+    }
+
+    if release_exists:
+        config["release"].update(
+            {
+                "ht": release_sites(data_type="genomes").ht(),
+                "select": [r for r in release_sites(data_type="genomes").ht().row],
+                "select_globals": [
+                    g for g in release_sites(data_type="genomes").ht().globals
+                ],
+            }
+        )
+    return config
+
+
+def custom_joint_faf_select(ht: hl.Table) -> Dict[str, hl.expr.Expression]:
+    """
+    Drop faf95 from 'grpmax'.
+
+    This annotation will be combined with the others from joint_faf's select in the config.
+    See note in `custom_freq_select` explaining why this field is removed.
+
+    :param ht: Joint FAF Hail Table.
+    :return: Select expression dict.
+    """
+    selects = {"joint_grpmax": ht.joint_grpmax.drop("faf95")}
+
+    return selects
+
+
+def custom_freq_select(ht: hl.Table) -> Dict[str, hl.expr.Expression]:
+    """
+    Drop faf95 from both 'gnomad' and 'non_ukb' in 'grpmax' and rename `gen_anc_faf_max` to `fafmax`.
+
+    These annotations will be combined with the others from freq's select in the config.
+
+    .. note::
+        - The faf95 field in the grpmax struct is the FAF of the genetic ancestry group with the largest AF (grpmax AF).
+        - The FAF fields within the gen_anc_faf_max struct contains the FAFs from the genetic ancestry group(s) with the largest FAFs
+        - These values aren't necessarily the same; the group with the highest AF for a variant isn't necessarily the group with the highest FAF for a variant
+        - The filtering allele frequencies that are used by the community are the values within the gen_anc_faf_max struct, NOT grpmax FAF, which is why we are dropping grpmax.faf95 and renaming gen_anc_faf_max
+
+    :param ht: Freq Hail Table
+    :return: Select expression dict.
+    """
+    selects = {
+        "grpmax": ht.grpmax.drop("faf95"),
+        "fafmax": ht.gen_anc_faf_max,
+    }
+
+    return selects
+
+
+def custom_in_silico_select(ht: hl.Table) -> Dict[str, hl.expr.Expression]:
+    """
+    Get in silico predictors from VEP for release.
+
+    This function currently selects only SIFT and Polyphen from VEP.
+
+    :param ht: VEP Hail Table.
+    :return: Select expression dict.
+    """
+    vep_in_silico = get_sift_polyphen_from_vep(get_vep().ht())
+    selects = {
+        "sift_max": vep_in_silico[ht.key].sift_max,
+        "polyphen_max": vep_in_silico[ht.key].polyphen_max,
+    }
+    return selects
+
+
+def custom_region_flags_select(ht: hl.Table) -> Dict[str, hl.expr.Expression]:
```

Comment:
Could also add a data_type parameter here and if "genomes" then add the interval QC. so then exomes and genomes can use the same function